min_heap_binary_tree.rb

```diff
@@ -80,4 +106,21 @@ def parent(index)
   def parents_index(index)
     index / 2
   end
+
+  def node_to_swap_with_two_children
```

Comment:
I pulled down the branch, and this `byebug` is not being hit.  We'll need a spec that is hitting this method.